source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +348,32 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
 
+        .. code-block:: python
+           :emphasize-lines: 4
 
+           key = client.encryption.keystore.find_one({ "_id": data_key})
```

Comment:
Also, I think there is an extra space after the opening curly brace, before `"_id"`

Reply:
Fixed:
- Added the creation of the client
- Removed the extra space before _id

Let me know what you think of the updated snippet? Note, I didn't include the definition of kms_providers because if I included the definition of kms_providers I would need to include the definition of local_master_key